packages/venia-concept/src/components/ProductImageCarousel/thumbnail.js

```diff
@@ -23,10 +25,10 @@ function Thumbnail(props) {
 
     const isDesktop = windowSize.innerWidth >= 1024;
 
-    function onClickHandlerWrapper() {
-        const { onClickHandler, itemIndex } = props;
+    const onClickHandlerWrapper = useCallback(() => {
```

Comment:
Conventionally,

- a callback received as a prop is called `onClick`
- a callback passed as a prop is called `handleClick`

```jsx
const { children, onClick } = props
const handleClick = useCallback(
  () => { onClick() },
  [onClick]
)

return (
  <button onClick={handleClick}>
    {children}
  </button>
)
```

Reply:
That makes sense :)